frontend/src/components/ProductCard.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ProductCard = () => {
+    return (
+        <ProductContainer>
+            <ProdImg>
+
+            </ProdImg>
+            <ProdInfo>
+                <ProdName>
+                    Coqueteleira 500ml
+                </ProdName>
+                <ProdDescription>
+                    Coqueteleira 500ml aço inox
+                </ProdDescription>
+                <OldPrice> 
+                    R$ 75,00
+                </OldPrice>
+                <ProdPrice>
+                    R$ 58,00
+                </ProdPrice>
+                <BtnAddCart>
+                    Adicionar ao carrinho
+                </BtnAddCart>
+            </ProdInfo>
+            
+        </ProductContainer>
+    )
+}
+
+const ProductContainer = styled.div`
+    font-family: 'Rubik', sans-serif;
+    //border: 1px solid black;
+    border-radius: 10px;
+    padding: 10px;
+    margin: 20px 0;
+    background-color: white;
+    box-shadow: 0px 5px 5px gray;
+
+`
+
+const ProdImg = styled.div`
+    background-color: gray;
+    width: 200px;
+    height: 200px;
+    margin-bottom: 10px;
+    border-radius: 5px;
+`
+
+const ProdInfo = styled.div`
+
+`
+
+const ProdName = styled.h1`
+    font-size: 1em;
+    font-weight: bold;
+    margin-bottom: 10px;
+`
+
+const ProdDescription = styled.p`
+    font-size: .8em;
+    margin-bottom: 10px;
+`
+
+const OldPrice = styled.div`
+    font-size: .8em;
+    margin-bottom: 10px;
+    text-decoration: line-through;
+` 
+
+const ProdPrice = styled.div`
+    font-size: 1.2em;
+    margin-bottom: 20px;
+`
+
+const BtnAddCart = styled.div`
+    color: white;
+    width: 100%; 
+    height: 40px;
+    background-color: #F2B705; // ${props => `${props.backgroundColor}`};
```

Comment:
Notei esse comentário aqui:

```js
// ${props => `${props.backgroundColor}`};
```

Talvez dê para remover esse comentário. Se tiver querendo pegar alguma prop do `styled-component` e não estiver conseguindo, só avisar que a gente ajusta isso.

```suggestion
    background-color: #F2B705;
```

Reply:
@Lukazovic esses comentários eram meus mas já corrigi pra próxima PR. Valeu!